src/syncData.test.ts

```diff
@@ -0,0 +1,90 @@
+import Redis from 'ioredis'
+import _ from 'lodash/fp.js'
+import {
+  initCollection,
+  initState as initRedisAndMongoState,
+  numDocs,
+} from 'mongochangestream-testing'
+import { MongoClient } from 'mongodb'
+import ms from 'ms'
+import { setTimeout } from 'node:timers/promises'
+import { describe, expect, test } from 'vitest'
+import debug from 'debug'
+
+// Output via console.info (stdout) instead of stderr.
+// Without this debug statements are swallowed by vitest.
+debug.log = console.info.bind(console)
+
+import { initSync, SyncOptions } from './index.js'
+
+const getConns = _.memoize(async () => {
+  // Redis
+  const redis = new Redis({ keyPrefix: 'testing:' })
+  // MongoDB source
+  const mongoSourceClient = await MongoClient.connect(
+    process.env.MONGO_SOURCE_CONN as string
+  )
+  const sourceDb = mongoSourceClient.db()
+  const sourceColl = sourceDb.collection('testing')
+  // MongoDB destination
+  const mongoDestintationClient = await MongoClient.connect(
+    process.env.MONGO_SOURCE_CONN as string
```

Comment:
Should this be `MONGO_DESTINATION_CONN`?

Reply:
It should. Great catch!